dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -661,6 +663,78 @@ def test_sample_size_passed_to_profile(self, *mocks):
         profiler._sampling_ratio = 0.2
         self.assertEqual(10000, update_mock.call_args[0][1])
 
+    @mock.patch('dataprofiler.profilers.profile_builder.'
+                'ColumnPrimitiveTypeProfileCompiler')
+    @mock.patch('dataprofiler.profilers.profile_builder.'
+                'ColumnStatsProfileCompiler')
+    @mock.patch('dataprofiler.profilers.profile_builder.'
+                'ColumnDataLabelerCompiler')
+    @mock.patch('dataprofiler.profilers.profile_builder.DataLabeler')
+    def test_index_overlap_resolved_for_update_profile(self, *mocks):
```

Comment:
the function name is kind of strange. Can we remove `resolved`?

Reply:
Done